resources/views/livewire/pagination/bootstrap.blade.php

```diff
@@ -0,0 +1,56 @@
+<div>
```

Comment:
This is pagination bootstrap template from Livewire but adapted to looks well on MailThief UI page.